src/App.module.css

```diff
@@ -1,22 +1,24 @@
 .header {
   grid-area: header;
+  height: 4rem;
```

Comment:
Do you need this height if you are already doing grid-template-rows below?

Reply:
you are right! no need. Will make this change shortly